webpack.config.js

```diff
@@ -1,5 +1,5 @@
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
```

Comment:
all the other strings in this file use single quotes. I'm not sure we'd want to change this